### ISSUE TITLE
VS Code plugin not saving to tldr file

### ISSUE THREAD

**Issue by eklem** [tldraw/tldraw#1866]
### What happened?

I have the tldraw plugin installed in VSCode and have some .tldr drawings in a GitHub repository. But when I check the .tldr file in a text editor it is empty. The .tldr framework is there, but no drawing content.

The drawing I've made:
<img width="1124" alt="Screenshot 2023-09-13 at 09 02 12" src="https://github.com/tldraw/tldraw/assets/236656/af3404d4-4b19-4615-8a46-a84d910bfa35">

Text view of the same file:
```JSON
{
  "tldrawFileFormatVersion": 1,
  "schema": {
    "schemaVersion": 1,
    "storeVersion": 4,
    "recordVersions": {
      "asset": {
        "version": 1,
        "subTypeKey": "type",
        "subTypeVersions": {
          "image": 2,
          "video": 2,
          "bookmark": 0
        }
      },
      "camera": {
        "version": 1
      },
      "document": {
        "version": 2
      },
      "instance": {
        "version": 17
      },
      "instance_page_state": {
        "version": 3
      },
      "page": {
        "version": 1
      },
      "shape": {
        "version": 3,
        "subTypeKey": "type",
        "subTypeVersions": {
          "group": 0,
          "embed": 4,
          "bookmark": 1,
          "image": 2,
          "text": 1,
          "draw": 1,
          "geo": 7,
          "line": 0,
          "note": 4,
          "frame": 0,
          "arrow": 1,
          "highlight": 0,
          "video": 1
        }
      },
      "instance_presence": {
        "version": 4
      },
      "pointer": {
        "version": 1
      }
    }
  },
  "records": [
    {
      "typeName": "shape",
      "id": null
    }
  ]
}
```

tldraw plugin info:
<img width="1109" alt="Screenshot 2023-09-13 at 09 03 05" src="https://github.com/tldraw/tldraw/assets/236656/11d55edf-dce4-44c4-861c-ceb963869692">

### How can we reproduce the bug?

* OSX v13.5.2
* VS Code v1.8.2.1
* tldraw plugin v2.0.11
* Create a .tldr file and draw something.
* Open the .tldr file in a text editor

I'm not sure which browser, I had Vivaldi as default OS browser, and now has Firefox. Same issue.

### What browsers are you seeing the problem on?

Chrome

### Contact Details

espen.klem@gmail.com

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by eklem** [comment]
Let me know if you need more info and I'll get it/post it.

**Comment by eklem** [comment]
Seems there was a bug, but fixed in tldraw extension for VSCode v2.0.12 released 2023-10-4, 20:50:02
